scripts/ci_install_dependency.sh

```diff
@@ -5,4 +5,8 @@ Install the dependency in CI.
 pip install --upgrade pip
 pip install -e "python[all]"
 pip install transformers==4.45.2 sentence_transformers accelerate peft
-pip install flashinfer -i https://flashinfer.ai/whl/cu121/torch2.4/ --force-reinstall
+git clone https://github.com/flashinfer-ai/flashinfer.git --recursive
+git reset --hard 32d9510d67187f1f3a379cce81302cdd15a557d2 # Revert to before PR https://github.com/flashinfer-ai/flashinfer/pull/609 merged
```

Comment:
We can use this for testing, not for release.

Reply:
Yes, this makes sense. This PR is mainly to test flashinfer for #2016 with flashinfer from source as it is failing all tests. We wanted to know where the issue stems from.